app/aws/s3.py

```diff
@@ -481,8 +472,16 @@ def get_phone_number_from_s3(service_id, job_id, job_row_number):
         )
         return "Unavailable"
 
-    phones = extract_phones(job, service_id, job_id)
-    set_job_cache(f"{job_id}_phones", phones)
+    phones = get_job_cache(f"{job_id}_phones")
+    if phones is None:
+        current_app.logger.debug("HAVE TO REEXTRACT PHONES!")
+        phones = extract_phones(job, service_id, job_id)
+        set_job_cache(f"{job_id}_phones", phones)
+        print(f"SETTING PHONES TO {phones}")
```

Comment:
Errant `print` statement! 🙂  Convert to another debug log line?

```suggestion
        current_app.logger.debug(f"SETTING PHONES TO {phones}")
```

Or just remove entirely?